src/main/java/dev/jbang/cli/Version.java

```diff
@@ -1,24 +1,46 @@
 package dev.jbang.cli;
 
+import java.io.IOException;
 import java.io.PrintWriter;
 
 import dev.jbang.Settings;
+import dev.jbang.util.Util;
+import dev.jbang.util.VersionChecker;
 
 import picocli.CommandLine;
 
 @CommandLine.Command(name = "version", description = "Display version info.")
 public class Version extends BaseCommand {
 
+	@CommandLine.Option(names = { "--check" }, description = "Check if a new version of jbang is available")
+	boolean checkForUpdate;
+
+	@CommandLine.Option(names = { "--update" }, description = "Update jbang to the latest version")
```

Comment:
Should we have a dedicated 'update' command for symmetry with install ?

Reply:
Perhaps, but that means we probably need to have a way to detect that newer versions of installed scripts are available (HTTP HEAD requests?).

Reply:
Actually, we don't have versions in scripts anyway. -_force can be used there to force update. 

Thus fine to have --update part of version for jbang